.github/workflows/verify.yml

```diff
@@ -73,10 +73,31 @@ jobs:
           exit 1
       - name: Install dependencies
         run: npm i --ignore-scripts --no-audit --no-fund
+      - name: Prepare icons
+        if: |
+          github.base_ref != 'master' &&
+          github.event_name == 'pull_request' &&
+          steps.changes.outputs.icons == 'true'
```

Comment:
Is it possible to define this condition as a variable?

The code repeats so many times in the file.